cdk/agents/lambda/bedrock-agentcore-deployer/deployer.py

```diff
@@ -0,0 +1,102 @@
+import json
+import boto3
+from botocore.config import Config
+from urllib.request import urlopen, Request
+
+client = boto3.client('bedrock-agentcore-control', config=Config(retries={'max_attempts': 5, 'mode': 'standard'}))
+
+def handler(event, context):
+    print(f"Event: {json.dumps(event)}")
+    
+    request_type = event['RequestType']
+    properties = event['ResourceProperties']
+    
+    try:
+        if request_type == 'Create':
+            return create_agent(properties, event, context)
+        elif request_type == 'Update':
+            return update_agent(properties, event, context)
+        elif request_type == 'Delete':
+            return delete_agent(properties, event, context)
+    except Exception as e:
+        return send_response(event, context, 'FAILED', {'Error': str(e)})
+
+def create_agent(properties, event, context):
+    agent_name = properties['AgentName']
+    image_uri = properties['ImageUri']
+    execution_role = properties['ExecutionRole']
+    
+    try:
+        response = client.create_agent_runtime(
+            agentRuntimeName=agent_name,
+            description=f'{agent_name} agent for Application Signals demo',
+            agentRuntimeArtifact={
+                'containerConfiguration': {
+                    'containerUri': image_uri
+                }
+            },
+            roleArn=execution_role,
+            networkConfiguration={
+                'networkMode': 'PUBLIC'
+            },
+            protocolConfiguration={
+                'serverProtocol': 'HTTP'
+            }
+        )
+        
+        agent_arn = response['agentRuntimeArn']
+        
+        return send_response(event, context, 'SUCCESS', {
+            'AgentArn': agent_arn,
+            'AgentName': agent_name
+        }, agent_arn)
+        
+    except Exception as e:
+        print(f"Agent creation failed: {str(e)}")
+        raise e
+
+def update_agent(properties, event, context):
+    return send_response(event, context, 'SUCCESS', {
```

Comment:
I am wondering why `update_agent` does nothing. If the implementation of nutrition agent or the primary agent changes, how do we do the update through CDK?